cypress/support/commands.ts

```diff
@@ -149,3 +149,55 @@ Cypress.Commands.add("getEmails", () => {
       return response;
     });
 });
+
+Cypress.Commands.add("seedCore", () => {
+  const authToken = window.localStorage.getItem("FLEET::auth_token");
+  cy.exec("bash ./tools/api/fleet/teams/create_core", {
+    env: {
+      TOKEN: authToken,
+      CURL_FLAGS: "-k",
+      SERVER_URL: Cypress.config().baseUrl,
+      // clear any value for FLEET_ENV_PATH since we set the environment explicitly just above
+      FLEET_ENV_PATH: "",
+    },
+  });
+});
+
+Cypress.Commands.add("seedBasic", () => {
+  const authToken = window.localStorage.getItem("FLEET::auth_token");
+  cy.exec("bash ./tools/api/fleet/teams/create_basic", {
+    env: {
+      TOKEN: authToken,
+      CURL_FLAGS: "-k",
+      SERVER_URL: Cypress.config().baseUrl,
+      // clear any value for FLEET_ENV_PATH since we set the environment explicitly just above
+      FLEET_ENV_PATH: "",
+    },
+  });
+});
+
+Cypress.Commands.add("seedFigma", () => {
+  const authToken = window.localStorage.getItem("FLEET::auth_token");
+  cy.exec("bash ./tools/api/fleet/teams/create_figma", {
+    env: {
+      TOKEN: authToken,
+      CURL_FLAGS: "-k",
+      SERVER_URL: Cypress.config().baseUrl,
+      // clear any value for FLEET_ENV_PATH since we set the environment explicitly just above
+      FLEET_ENV_PATH: "",
+    },
+  });
+});
+
+Cypress.Commands.add("addUser", (username, options) => {
+  options ||= {};
+  let { password, email, globalRole } = options;
+  password ||= "test123#";
+  email ||= `${username}@example.com`;
+  globalRole ||= "admin";
```

Comment:
@martavis can you let me know whether this is expected style for handling such optional arguments? Or if not, what is the better/more standard way?

Reply:
`options` can be defaulted in line 192 inside the parameter declaration. Otherwise, looks good.